drs/Dockerfile

```diff
@@ -62,7 +62,7 @@ RUN git config --global user.email "you@example.com" && \
     git pull && \
 	git checkout tags/${TAG}
```

Comment:
Suggestion: the same recipe as in BentoV2 to allow for a branch name to be effective when the tag is not set, can be used here. It would avoid the config/checkout/pull commands as well. 
```
RUN git clone --depth 1 -b "${TAG:-$BRANCH}" "${REPO}"
```
(--depth 1 is facultative: it limits the download to the last state of the branch, not the whole git history)

Reply:
ah yes, thanks!